LEAF_Request_Portal/templates/print_form.tpl

```diff
@@ -461,42 +484,296 @@ function cancelRequest() {
             		alert(response);
             	}
             },
+            error: function(){ console.log('There was an error canceling the request!'); },
             cache: false
 		});
 	});
 	dialog_confirm.show();
 }
 
 function changeTitle() {
-	dialog.setContent('Title: <input type="text" id="title" style="width: 300px" name="title" value="<!--{$title|escape:'quotes'}-->" /><input type="hidden" id="CSRFToken" name="CSRFToken" value="<!--{$CSRFToken}-->" />');
-  //ie11 fix
-  setTimeout(function () {
+    dialog.setContent('Title: <input type="text" id="title" style="width: 300px" name="title" value="<!--{$title|escape:'quotes'}-->" /><input type="hidden" id="CSRFToken" name="CSRFToken" value="<!--{$CSRFToken}-->" />');
     dialog.show();
-  }, 0);
     dialog.setSaveHandler(function() {
         $.ajax({
-        	type: 'POST',
-        	url: 'api/form/<!--{$recordID|strip_tags}-->/title',
-        	data: {title: $('#title').val(),
+            type: 'POST',
+            url: 'api/form/<!--{$recordID|strip_tags}-->/title',
+            data: {title: $('#title').val(),
                 CSRFToken: '<!--{$CSRFToken}-->'},
-        	success: function(res) {
-        		if(res != null) {
-        			  $('#requestTitle').empty().html(res);
-        		}
+            success: function(res) {
+                if(res != null) {
+                      $('#requestTitle').empty().html(res);
+                }
                 dialog.hide();
-        	}
+            },
+            error: function(){ console.log('There was an error changing the title!'); }
         });
     });
 }
 
+/**
+ *
+ * @param {object} indicators
+ * @returns {array}
+ */
+function getChildrenIndicatorIDs(indicators) {
```

Comment:
You check to see if it's null, but you don't verify that it's an object with the needed key-values.  How will javascript react if "indicators" is "plaid"?